### DISCUSSION TITLE
move non-app endpoints (e.g. /health) to different port

### DISCUSSION THREAD

**Discussion by kenyee** [quarkusio/quarkus#23848] [Q&A] [answered]
Has anyone found a workaround to this?
https://github.com/quarkusio/quarkus/issues/13602

Trying to figure out how to move the /health endpoint to a different port or even to mirror it to a different port.

[a comment by an automated account is not shown]

**Comment by kenyee** [comment]
ended up moving the scanning of these endpoints to use the same http port so not an issue for me any more.

Surprised there's no option for this though.

**Comment by xstefank** [comment]
I wasn't aware of this issue and Ken left RH to pursue other challenges. I'll take a look at what can be done.

**Comment by kenyee** [comment]
Sorry for the ping @xstefank 
was wondering if you had a chance to look at this?
Seems like this is hitting quite a few folks in that thread...

**Comment by xstefank** [comment]
@kenyee sorry, not yet. I will move this up to look at it this week.

**Comment by kenyee** [comment]
Thanks @xstefank  :-)

**Reply by kenyee** [reply]
@xstefank any chance you've had a look at it?  My company decided to clamp down on this requirement so I can't work around it by running on the same port any more :-(

**Reply by kenyee** [reply]
has anyone found a different workaround to run two instances of quarkus serving different ports and maybe proxying a port over?
EDIT: FYI, using nginx in the same container to proxy a port works...

**Reply by xstefank** [reply]
@kenyee yes, we discussed how to do it on zulip and I've started with implementation. But I am busy on a different front now, hopefully, soon I will make a PR.

**Comment by xstefank** [comment, marked as answer]
This is now possible thanks to https://github.com/quarkusio/quarkus/pull/30506! 

`quarkus.management.enabled=true`

...Listening on: http://localhost:8080. Management interface listening on http://0.0.0.0:9000.
